dlp/api/Snippets.Tests/resources/test.txt

```diff
@@ -1 +1 @@
-My name is Robert Frost
+My name is Robert Frost and email is test@gmail.com
```

Comment:
Please use an email address from: https://developers.google.com/style/examples#example-ip-addresses

Reply:
Changed to alex@example.com